File: api/src/main/java/net/blueshell/api/service/EventPictureService.java
package net.blueshell.api.service;

import net.blueshell.api.base.BaseModelService;
import net.blueshell.api.model.EventPicture;
import net.blueshell.api.repository.EventPictureRepository;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.context.ApplicationEventPublisher;
import org.springframework.stereotype.Service;

@Service
public class EventPictureService extends BaseModelService<EventPicture, Long, EventPictureRepository> {

    @Autowired
    public EventPictureService(EventPictureRepository repository, ApplicationEventPublisher events) {
        super(repository);
    }
}

File: api/src/main/kotlin/net/blueshell/api/service/event/EventPictureService.kt
package net.blueshell.api.service.event

import net.blueshell.api.model.event.EventPicture
import net.blueshell.api.repository.event.EventPictureRepository
import net.blueshell.api.service.base.BaseModelService
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.context.ApplicationEventPublisher
import org.springframework.stereotype.Service

@Service
class EventPictureService @Autowired constructor(
    repository: EventPictureRepository,
    events: ApplicationEventPublisher
) : BaseModelService<EventPicture, EventPicture.Id, EventPictureRepository>(repository)
